src/applications/disability-benefits/526EZ/content/confirmation-page.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import moment from 'moment';
+
+import siteName from '../../../../platform/brand-consolidation/site-name';
+import {
+  successMessage,
+  checkLaterMessage,
+  errorMessage,
+} from '../content/confirmation-poll';
+import { get4142Selection } from '../helpers';
+
+const privateRecordReleaseContent = (
+  <div>
+    <p>
+      <strong>
+        If you need us to get your private medical records from your doctor,
+      </strong>{' '}
+      you’ll need to fill out an Authorization to Disclose Information to the VA
+      (VA Form 21-4142).
+    </p>
+    <p>
+      <a
+        href="https://www.vba.va.gov/pubs/forms/VBA-21-4142-ARE.pdf"
+        target="_blank"
+      >
+        Download VA Form 21-4142
+      </a>
+      .
+    </p>
+    <p>Please print the form, fill it out, and send it to:</p>
+    <p className="va-address-block">
+      Department of Veterans Affairs
+      <br />
+      Claims Intake Center
+      <br />
+      PO Box 4444
+      <br />
+      Janesville, WI 53547-4444
+    </p>
+  </div>
+);
+
+const template = (props, title, content, submissionMessage) => {
+  const { fullName, disabilities, submittedAt } = props;
+  const { first, last, middle, suffix } = fullName;
+
+  const selected4142 = get4142Selection(disabilities || []);
+
+  const renderableContent =
+    typeof content === 'string' ? <p>{content}</p> : content;
+
+  return (
+    <div>
+      <h3 className="confirmation-page-title">{title}</h3>
+      {renderableContent}
+      {selected4142 && privateRecordReleaseContent}
+
+      <div className="inset">
+        <h4>
+          Disability Compensation Claim{' '}
+          <span className="additional">(Form 21-526EZ)</span>
+        </h4>
+        <span>
+          For {first} {middle} {last} {suffix}
```

Comment:
🤔 don't think this is _really_ a part of your PR but just thought I'd point out that the name spacing will be weird if the veteran doesn't have a middle name.

Reply:
I think it actually won't be. Pretty sure extra spaces in HTML are all collapsed into a single space unless it's within a `pre` block.

Reply:
hm, interesting.